src/modules/usersScore.js

```diff
@@ -0,0 +1,49 @@
+class UserScore {
+  constructor(user, score) {
+    this.user = user;
+    this.score = score;
+  }
+
+    playersData = [];
+
+    url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/QE8nBkXmdfd1YoxrlAjT/scores/';
+
+    getData = () => {
+      const scores = document.getElementById('scores-container');
+      scores.innerHTML = this.playersData.map((elem, index) => `<p class=${index % 2 !== 0 ? 'row-bg' : ''} >${elem.user}: <span>${elem.score}</span></p>`).join('');
+    }
+
```

Comment:
- [ ] When I Click on refresh button the score should be updated with new values only. Now this action cause duplication of same values.Please fix this.
![Screenshot from 2022-05-19 13-42-38](https://user-images.githubusercontent.com/65860678/169286522-4c8129cc-6a78-499f-bc78-e7eeb547cdf3.png)

Reply:
Any suggestions to fix that?